src/main/java/org/example/homework10/EmployeeProcessing/Main.java

```diff
@@ -0,0 +1,71 @@
+package org.example.homework10.EmployeeProcessing;
+
+import java.lang.reflect.Field;
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) throws ParseException {
+
+        Employee e1 = new Employee("Minnie", "Mouse", "01/02/2015");
+        Employee e2 = new Employee("Mickie", "Mouse", "05/08/2000");
+        Employee e3 = new Employee("Daffy", "Duck", "11/02/2011");
+        Employee e4 = new Employee("Daisy", "Duck", "05/03/2013");
+        Employee e5 = new Employee("Goofy", "Dog", "23/07/2020");
+
+        List<Employee> list = new ArrayList<>(Arrays.asList(e1, e2, e3, e4, e5));
+
+        class AdditionalInfo {
```

Comment:
better put local class to `printOrderedList `
and you should use it with Employee field inside